client/src/pages/mainPage/components/home/Home.css

```diff
@@ -0,0 +1,53 @@
+/* Hero Styles */
+.booster__hero{
+	background-image:url("../energiser-Imgs/Booster-hero-img.png");
+	/* background-position: center;
```

Comment:
Try and not merge commented out code to main branch

Reply:
I'll fix it now